docs/orchestrating-elastic-stack-applications/apm-server.asciidoc

```diff
@@ -75,9 +76,17 @@ apm-server-quickstart-apm-server-69b447ddc5-fflc6   1/1     Running   0
 [id="{p}-apm-advanced-configuration"]
 == Advanced configuration
 
+This section covers the following topics:
+
+** <<{p}-apm-agent-central-configuration,Use APM Agent central configuration>>
+** <<{p}-apm-customize-configuration,Customize the APM Server configuration>>
+** <<{p}-apm-secure-settings,APM Secrets keystore for secure settings>>
+** <<{p}-apm-existing-es,Reference an existing Elasticsearch cluster>>
```

Comment:
The overridden link text in some of the links don't match the actual title of the section they are linking to. Removing the explicit link text would keep them in sync.

```suggestion
** <<{p}-apm-agent-central-configuration>>
** <<{p}-apm-customize-configuration>>
** <<{p}-apm-secure-settings>>
** <<{p}-apm-existing-es>>
```

Reply:
Nice! Thanks @charith-elastic